README.md

```diff
@@ -77,6 +77,14 @@ export LIBVA_DRIVERS_PATH=<path-contains-iHD_drv_video.so>
 export LIBVA_DRIVER_NAME=iHD
 ```
 
+## Docker Image
+Docker images are fully qualified by media stack suites. Latest image is targeting to Ubuntu 18.04. You can get it by
```

Comment:
This paragraph appears "suddenly" saying something about docker images:). I think it should be rewritten. Actually start with something like: "To simplify media stack installation you can use docker images. They are located <blah-blah-blah>".  So, first introduce what you are talking about and only when say anything about their "qualification".

Reply:
Sure. thanks for your comment.